src/components/Single-review.jsx

```diff
@@ -29,7 +42,9 @@ export default function SingleReview() {
         <p className="user">User: {singleReview.owner}</p>
         <img className="img" src={singleReview.review_img_url} alt="" />
         <p className="body">{singleReview.review_body}</p>
-        <p>Votes: {singleReview.votes}</p>
+        <button onClick={() => handleVotes(review_id, 1)}>Upvote</button>
+        <button onClick={() => handleVotes(review_id, -1)}>Downvote</button>
+        <p>Votes: {singleReview.votes + reviewVotes}</p>
```

Comment:
conditionally render a p tag here to tell user the vote failed if the state is true